package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nativescript-starter-kits",
-  "version": "0.1.16",
+  "version": "0.1.17",
```

Comment:
No need to bump the version here, because npm version patch && npm publish handles this scenario

Reply:
Great, I've didn't knew it!